### PR TITLE
Make bubble chart clickable

### PR DESCRIPTION
**Pull request by kadamwhite**
@iros Please let me know if you see a cleaner way to do any of this

clickable-chart

### CODE REVIEW

File: src/components/EmojiBubbleChart.js [hunk 1]
```diff
@@ -94,30 +97,51 @@ class EmojiBubbleChart extends PureComponent {
...
 
     const scaler = 1.5;
+    const zoomScaler = 1.6;
     const sumValues = d3.sum(root.children, d => d.value);
 
     // == render emoji ==
     const emojiBinding = parent.selectAll('div.emoji')
       .data(root.children, d => d.id);
 
+    console.log(root.children);
```
Comment (iros) [inline src/components/EmojiBubbleChart.js:107]: Prob want to remove the console.log?

File: src/components/EmojiBubbleChart.js [hunk 2]
```diff
@@ -94,30 +97,51 @@ class EmojiBubbleChart extends PureComponent {
...
-      .style('width', '0px')
-      .style('height', '0px');
+      .style('left', d => `${d.x}px`);
 
     enteringEmoji.merge(emojiBinding)
       .each(function renderEmoji() {
         twemoji.parse(this, icon => emojiSVGUrl(icon));
         return this;
       })
+      .on('mouseover', function onMouseover(d) {
```
Comment (iros) [inline src/components/EmojiBubbleChart.js:122]: How does this behave on mobile? Legit don't know so mostly curious.

**Review by iros — APPROVED** [review]
Looks good. a few small comments/questions

File: src/components/EmojiBubbleChart.js (hunk 1 above)
Comment (kadamwhite) [reply src/components/EmojiBubbleChart.js:107]: 👍

File: src/components/EmojiBubbleChart.js (hunk 2 above)
Comment (kadamwhite) [reply src/components/EmojiBubbleChart.js:122]: Going to merge, then I will test once I'm at the office; I brought a mini device lab with me :)

**Pull request merged**